proofs/rustverifier.go

```diff
@@ -81,8 +81,9 @@ func (rp *RustVerifier) VerifyPoST(req VerifyPoSTRequest) (VerifyPoSTResponse, e
 	defer elapsed("VerifyPoST")()
 
 	// flattening the byte slice makes it easier to copy into the C heap
-	flattened := make([]byte, 32*len(req.CommRs))
-	for idx, commR := range req.CommRs {
+	commRs := req.SortedCommRs.Values()
+	flattened := make([]byte, 32*len(commRs))
```

Comment:
(Non-blocking) This isn't your change, but I'd really prefer if we just used the actual type for the first argument to `make` instead of `[]byte` and multiplying the length by 32. This could be an issue if commRs ever change to a type that isn't 32 bytes in size.

Reply:
> This could be an issue if commRs ever change to a type that isn't 32 bytes in size.

We have to flatten our slice of fixed-length arrays into a byte slice in order to work with `C.CBytes`. 

Also, `CommR` is a [specified type](https://github.com/filecoin-project/specs/blob/99f42e76e9b3e4f309578dc1a2f1c6e4bfabfb48/proofs.md) and will likely not change from a 32-byte array.